INFO     root:conftest.py:184 === Starting auth_token Test ===
INFO     root:conftest.py:185 API Response: 200 | {"kind":"identitytoolkit#VerifyCustomTokenResponse","idToken":"eyJhbGciOiJSUzI1NiIsImtpZCI6IjNhM2JkODk4ZGE1MGE4OWViOWUxY2YwYjdhN2VmZTM1OTNkNDEwNjgiLCJ0eXAiOiJKV1QifQ.eyJhZ3JlZW1lbnRzU2lnbmVkIjpmYWxzZSwiZW1haWxWZXJpZmllZCI6dHJ1ZSwibWVtYmVyQmlydGhEYXRlIjoiMDMvMDgvMTk5MSAwMDowMDowMCIsIm1lbWJlcklkIjozODI5LCJtZW1iZXJGaXJzdE5hbWUiOiJRQSIsIm1lbWJlckxhc3ROYW1lIjoiTmVyZCIsIm1lbWJlckRpc3BsYXlOYW1lIjoiTmVyZENvcmUtUUEiLCJtZW1iZXJOYW1lIjoiTmVyZCBDb3JlIiwibWVtYmVyTnVtYmVyIjoiMDAzODI5IiwibWVtYmVyUGhvbmUiOiIrMTMxMDU1NTU1NTUiLCJtZW1iZXJUaW1lWm9uZSI6IiswNzowMCIsInBlcm1pc3Npb25zIjpbXSwicm9sZXMiOltdLCJ1c2VySWQiOiI2NDJlZWZhNjc1ZGU3MDdjNDRhZmM4NDAiLCJvcHRJblNoYXJlTmFtZSI6ZmFsc2UsImF1dGhlbnRpY2F0b3JFbmFibGVkIjpmYWxzZSwiaXNzIjoiaHR0cHM6Ly9zZWN1cmV0b2tlbi5nb29nbGUuY29tL2NvcmUtZGV2LTliNTRiIiwiYXVkIjoiY29yZS1kZXYtOWI1NGIiLCJhdXRoX3RpbWUiOjE3MDIxOTE5MTgsInVzZXJfaWQiOiJvQ0RSbHNNOVMzWjdiejBXNzMyYlBUcWlGUUgzIiwic3ViIjoib0NEUmxzTTlTM1o3YnowVzczMmJQVHFpRlFIMyIsImlhdCI6MTcwMjE5MTkxOCwiZXhwIjoxNzAyMTk1NTE4LCJlbWFpbCI6Im5lcmRjb3JlX3FhdGVhbSsxQG5lcmR1bml0ZWQuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnsiZW1haWwiOlsibmVyZGNvcmVfcWF0ZWFtKzFAbmVyZHVuaXRlZC5jb20iXX0sInNpZ25faW5fcHJvdmlkZXIiOiJjdXN0b20ifX0.iY4_X4yzLSe-zLtIDJZv8FTuKF2YuehMKwy07u8lTVoqK-4LwjhfBhANyC9eZO8lswsja9kbzs-oxXA-xG3qpGwoWAg_3W0eZ6t8CCzfFgKMH4U1YhboTOpz7TgWvwdBv-l8Xw4-cCf6x8Ox4-9LiPF7ABXvqlpJT_Mlp9sVuUpchVaJ9wr1HsdesN6FUP87mXBY8I4TPNMCTuJ0yS1mSksQP0FTlvh5RWqVG-oyKuSudBdw3ql2NFppUebnUECASPJiV5pl2f4_bTq3JLk2MEmqrBZGZFhf5A8wvO89jWCwaoBpzfEDIqQdZK-V2WSLxIiyYf4GrHzebFmIN14JPw","refreshToken":"AMf-vBxTbyRyvYfp9gxWgQXV1zfbW3Cr3Lep90c5SAEPNtyLy2fBQu-FmPrhillHj96_aJY8Us6nhvo1aZYQ2ITdr1BaDWZZUHyet572nY-Mx5gtYLwqZg5pnLGVwckW9vXExp6wn4MG6huE9OD6RJjvlons-xSY_2NEXxJGKjPCKTpSgQVJ5b-HEINvedNkI7wqy0FoGC7QEJ0k40P53CcLermkNziM3sM7UXYEJYgPwagmnr3L4YYXY8gyTggiv_R7tkyxsxd8Z5mA2aRIHGre-2r6CD5Npow44nrnPNSvoGt6mzsgaQ1C0jp09_28YjHmT_gL7VLUM4ZrKcxHSf2KchqZjQ2jRbbGQijeILo3SQQ0HF2rwDnf_PHIWRWkRFr5-h4_QZF2HgVcTsjrdbZQHdQzPZHz-55tGAEkkqLL0mgsA-JmoNKskb1EGLv4Xf6_JJrSvsd9jlvTlbU0dMKiU1CUlkduCFSkWPMu09tcCKo-YODAQllfTqZO-GIyx-yGr5kp0Mn9d7cvhPEO7gO3nuyMPmQpKQbyEb0swlotNYQa6FfIlPbGJxhUQzcIXpX6OTUp7JJUH0FJC7EjyXBBKLvXIb75tfTwQpCB7eCMMb1T_6xy6elvSXX2UAGCYMYB5rsRE0P9HYcrr6i0EZKoGej5KNoa49qDz8Zcb__i_JDlBvMcJt0XnsT0RX0TD9E9O6Ad34SWdDmUtW2ZU_H9juk7qJiHmA","expiresIn":"3600","isNewUser":false}
INFO     root:conftest.py:195 Test 'auth_token' SUCCESS.
INFO     root:conftest.py:196 === Ending auth_token Test ===
INFO     root:conftest.py:184 === Starting test_get_a_role Test ===
INFO     root:conftest.py:185 API Response: 401 | 
INFO     root:conftest.py:191 Test 'test_get_a_role' FAILED.
ERROR    root:conftest.py:192 API response status code: 401
assert 401 in [200, 201, 202, 203, 204, 205]
 +  where 401 = <Response [401]>.status_code
Traceback (most recent call last):
  File "/home/runner/work/qa-nerd-core-backend/qa-nerd-core-backend/conftest.py", line 189, in api_response_conditions
    assert response.status_code in allowed_status_codes, f"API response status code: {response.status_code}"
AssertionError: API response status code: 401
assert 401 in [200, 201, 202, 203, 204, 205]
 +  where 401 = <Response [401]>.status_code